### DISCUSSION TITLE
Allow Hidden Categories

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#149] [Feature ideas]
A question was asked on Reddit about how to hide a category.  I know categories can be deleted, and the activity from it moved to a different category, but sometimes there are categories that you don't necessarily want to move the activity anywhere else, but you also don't want to use any more.  So you'd like it to not clutter up the budget.  

Consider this an official feature request for hidden categories on behalf of the reddit user who asked the question.

(my interim suggestion was to create a "hidden" category group and move it/them to that group.  It can be collapsed and moved to the bottom so it's rarely seen, and doesn't need to be interacted with.)

**Comment by smathson** [comment]
> (my interim suggestion was to create a "hidden" category group and move it/them to that group. It can be collapsed and moved to the bottom so it's rarely seen, and doesn't need to be interacted with.)

I currently use this workaround as well but first-class "hidden" or "archive" category support would be a nice addition.

**Reply by Rick-Thomas** [reply]
One  thing to bare in mind with Hidden categories is that they loose "contact" with their original Group, and become part of the Hidden category group...   When you look at historical data you must manually relate these to the original Group...   Example: if I hide the "Home:Garden Maintenance", it now becomes "Hidden:Garden Maintenance" and in the historical analysis one must manually adjust those expenses to the original one...   

Some way of maintaining this link so's to have reports reflect them as they should would be and ideal solution...
